applications/MappingApplication/custom_mappers/coupling_geometry_mapper.h

```diff
@@ -340,6 +351,28 @@ class CouplingGeometryMapper : public Mapper<TSparseSpace, TDenseSpace>
         }
     }
 
+    void CreateLinearSolver()
+    {
+        bool is_linear_solver_specified = false;
+        if (mMapperSettings.Has("linear_solver_settings"))
+        {
+            if (mMapperSettings["linear_solver_settings"].Has("solver_type"))
+            {
+                is_linear_solver_specified = true;
+                mpLinearSolver = LinearSolverFactory<TSparseSpace, TDenseSpace>().Create(mMapperSettings["linear_solver_settings"]);
+            }
+        }
+        if (!is_linear_solver_specified)
+        {
+            // TODO - replicate 'get fastest solver'
```

Comment:
@philbucher 'getfastestlinearsolver' is only implemented on the python level atm. I think falling back to skyline for the time being is fine - if people want faster solvers they can still specify it.

Reply:
that default lin solver should be in the defaults of the mapper

I will do this in a separate PR, can be updated since #7290